src/crimson/osd/objclass.cc

```diff
@@ -187,11 +177,8 @@ int cls_cxx_replace(cls_method_context_t hctx,
     OSDOp top{CEPH_OSD_OP_TRUNCATE};
     top.op.extent.offset = 0;
     top.op.extent.length = 0;
-    try {
-      reinterpret_cast<ceph::osd::OpsExecuter*>(hctx)->execute_osd_op(top).get();
-      return 0;
-    } catch (ceph::osd::error& e) {
-      return -e.code().value();
+    if (const auto ret = execute_osd_op(hctx, top); ret < 0) {
+      return ret;
```

Comment:
we need to return the good news as well.

Reply:
Oops, fixed. Thanks!